src/doc/trpl/raw-pointers.md

```diff
@@ -0,0 +1,122 @@
+% Raw Pointers
+
+Rust has a number of different smart pointer types in its standard library, but
+there are two types that are extra-special. Much of Rust’s safety comes from
```

Comment:
Which two? `&` and `&mut`? or `*const` and `*mut`? It's seems to me that the raw pointer types are the least special (at least when compared to C/C++/other languages).

Also, I wouldn't call the raw pointer types "smart pointers".